src/pages/guides/sdk/sdk_signature_verification.md

```diff
@@ -6,7 +6,76 @@ title: Signature Verification for Events | Adobe I/O Events SDK
 
 Your webhook URL must be accessible from the open internet, however this means third-party actors can send forged requests, tricking your application into handling fake events.
 
-To prevent this from happening, Adobe I/O Events adds an `x-adobe-signature` header to each POST request it does to your webhook URL, which allows you to verify that the request was really made by Adobe I/O Events.
+To prevent this from happening, Adobe I/O Events has a robust event validation process in place as defined below that allows users to secure their webhook. 
+
+### Digital Signatures for Security Verification
+For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
+
+- Adobe I/O Events sends an additional field of `recipient_client_id` as part of your event payload.
+- The event payload is signed digitally using a fixed public/private key pair generated by Adobe I/O Events. The digital signature is sent as a webhook request header.
+- Adobe I/O Events sends the relative path of public key, which is served from our fixed Adobe domain [static.adobeioevents.com](https://static.adobeioevents.com), as webhook request headers.
+
+I/O Events sends 2 digital signatures as webhook request headers and they are available via the header fields
+`x-adobe-digital-signature-1` and `x-adobe-digital-signature-2` respectively.
+
+I/O Events also sends relative paths of 2 public keys corresponding to the private keys used to generate the digital signatures. These public keys are publicly accessible using our Adobe domain [static.adobeioevents.com](https://static.adobeioevents.com) and the webhook request header fields `x-adobe-public-key1-path` and  `x-adobe-public-key2-path` respectively.
+
+This SDK method allows you to pass the digital signature headers, public keys relative paths received and the JSON payload delivered to the webhook to check its authenticity. The JSON payload contains the `recipient-client-id` which can be matched against your own webhook registration client id passed to this SDK method. The method returns `true` if the calculated signature matches that of the header, otherwise it returns `false`. 
+
+This SDK api can be used in any digital signature verification implementation for your consumer app to verify the authenticity of events coming from Adobe I/O Events.  
+
+## Method
+
+```javascript
+verifyDigitalSignatureForEvent(event, recipientClientId, [signatureOptions]) ⇒ boolean
+```
+| Param | Type | Description |
+| --- | --- | --- |
+| event | <code>object</code> | JSON payload delivered to the registered webhook URL |
+| recipientClientId | <code>string</code> | Target recipient client id retrieved from the Adobe I/O Console integration |
+| [signatureOptions] | [<code>SignatureOptions</code>](#SignatureOptions) | Map of digital signature header fields defined in SignatureOptions |
+
+### SignatureOptions : <code>object</code>
```

Comment:
This is causing issues while being loaded in the side menu. We can change this to:
```suggestion
### SignatureOptions: `object`
```
![signature-options](https://user-images.githubusercontent.com/6910192/155079722-6598f0a8-ac48-4ee3-ab15-ed8f5b9a8610.png)